docs/developer-docs/latest/development/custom-fields.md

```diff
@@ -198,6 +223,47 @@ Each object in the `items` array can contain the following parameters:
 | `intlLabel`     | Translation for the label of the input                             | [`IntlObject`](https://formatjs.io/docs/react-intl/) |
 | `type`          | Type of the input (e.g., `select`, `checkbox`)                     | `String`                                             |
 
+::: details "type" accepts the following inputs:
+
+```javascript
+[
+  'text',
+  'checkbox',
+  'checkbox-with-number-field',
+  'boolean-radio-group',
+  'select-number',
+  'radio-group',
+  'select',
+  'select-default-boolean',
+  'select-date',
+  'textarea-enum',	
+]
+```
+::: 
+
+You can also set a `defaultValue` for options , below I use the select-number component and provide a `defaultValue` of `3`.
```

Comment:
I can't find "select-number" in the code example 🤔 What do you mean here?

Reply:
Something like this but maybe phrased better...

```suggestion
A setting item also accepts a `defaultValue`. This allows you to initialize the value of the input. For example below I use the select-number component input and provide a `defaultValue` of `3`. When accessing the input in the CTB I should see the input pre-filled with `3`
```